README.md

```diff
@@ -1,8 +1,33 @@
-# BA
-My Bachelorthesis Repository
+# A cuboid muscle model
 
-**Content:**
+## Setup
+- A dummy cuboid muscle geometry. 
+- Solver comprises hyperelasticity solver for the prestretch and both mechanics solver + fastmonodomain solver for the contraction. 
+- It uses the CellML model "hodgkin_huxley-razumova".
+- No preCICE involved. 
 
--cuboid_muscle
+### How to build?
+Follow OpenDiHu's documentation for installation, then run 
+```
+mkorn && sr
+```
+For a debug build, look into the documentation. 
 
--BayesOpt
+### How to run?
+To run a single simulation of stretching a muscle with a certain force and then contract it, go to build_release and run:
+
+```
+./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py incompressible_mooney_rivlin 10.0
```

Comment:
you can directly run it with 
`./muscle_contraction_with_prestretch ../settings_contraction_with_prestretch.py 10.0`

Reply:
But then you can't specify the force. Line 28 in settings_contraction_with_prestretch.py requires you to add a scenario name, else you can't hand over the force value.